packages/dm-core-plugins/src/form/fields/ObjectField.tsx

```diff
@@ -174,36 +175,36 @@ export const ContainedAttribute = (props: TContentProps): JSX.Element => {
               onAdd={() => setIsDefined(true)}
             />
           ))}
-        {hasOpen && isDefined && (
-          <OpenObjectButton
-            viewId={namePath}
-            idReference={idReference}
-            namePath={
-              attributePath && attributePath.length > 1
-                ? `${attributePath[1]}.${namePath}`
-                : namePath
-            }
-          />
-        )}
-        {!hasOpen && isDefined && (
-          <>
-            {uiRecipe && uiRecipe.plugin !== 'form' && (
-              <EntityView
-                idReference={`${idReference}.${namePath}`}
-                type={type}
-                onOpen={onOpen}
-              />
-            )}
-            {(uiRecipe === undefined ||
-              (uiRecipe && uiRecipe.plugin === 'form')) && (
-              <AttributeList
-                namePath={namePath}
-                config={uiRecipe?.config}
-                blueprint={blueprint}
-              />
-            )}
-          </>
-        )}
+        {isDefined &&
+          (hasOpen && !uiAttribute?.isInline ? (
```

Comment:
I might just be having a really hard time reading this, but should the two checks not be combined with the closing parentheses ending after isInline? 
```
(hasOpen && !uiAttribute?.isInline) ? (
  // content
)
```

Reply:
So it was definitely me having a hard time reading it 😆 Doesn't mean that the code is necessarily too complex though! If I had opened it in vs code I would've had a much easier time comprehending what it said - I just wanted to double check because this kind of splitting is usually what prettier just does automatically sometimes 👍🏻